docs/openapi-fetch/api.md

```diff
@@ -88,9 +88,11 @@ const client = createClient({
 | form (exploded)          | `/users?role=admin&firstName=Alex`                   |
 | **deepObject (default)** | `/users?id[role]=admin&id[firstName]=Alex`           |
 
-> [!NOTE]
```

Comment:
I incorrectly thought Vitepress changed their syntax, but this is just an alternate form of the same thing. The `:::` is required for a lot of Vitepress markdown features so I’ll stick with that.